contracts/BSM/Investors/AMM/interfaces/ISovrynSwapNetwork.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface ISovrynSwapNetwork {
+    /**
+    @return The outAmount of the swap
+    */
+    function convertByPath(
+        IERC20[] calldata _path,
+        uint256 _amount,
+        uint256 _minReturn,
+        address _beneficiary,
+        address _affiliateAccount,
+        uint256 _affiliateFee
+    ) external payable returns (uint256);
+
+    /**
+    @return the rate of the swap
+     */
+    function rateByPath(IERC20[] calldata _path, uint256 _amount) external view returns (uint256);
+
+    /**
+    token0 - anchor - token1 - anchor - token2
```

Comment:
maybe we should add here the link explaining this comment because it means nothing this way

Reply:
Sure why not